--- intlfonts-1.2.1/Makefile.in.orig	2008-11-23 05:26:10.126934224 +0100
+++ intlfonts-1.2.1/Makefile.in	2008-11-23 05:28:18.618103320 +0100
@@ -38,7 +38,7 @@
 	  rm -f ${fontdir}/temp ; \
 	fi
 	@for dirs in ${SUBDIRS} ${SUBDIRS_X} ${SUBDIRS_BIG}; do \
-	  destdir=`echo $${dirs} | sed 's/\([a-zA-Z]*\)\.BIG/\1/'` ; \
+	  destdir=`echo $${dirs} | sed 's/\([a-zA-Z]*\)\.\(BIG\|X\)/\1/'` ; \
 	  if [ ! -d ${fontdir}/$${destdir} ] ; then \
 	    $(INSTALL) -d ${fontdir}/$${destdir} ; \
 	  fi ; \
